### DISCUSSION TITLE
Welcome to StarCitizen-Localization Discussions!

### DISCUSSION THREAD

**Discussion by Dymerz** [Dymerz/StarCitizen-Localization#14] [Announcements]
## 👋 Welcome!
Welcome to the GitHub discussion for the Star Citizen localization project! 
We're delighted to have you here to discuss the specifics of our translation mission. During this discussion, we will explore key elements such as the content to be translated, target languages, quality standards, and collaboration methods. Your contributions and ideas are crucial to moving this project forward, so please feel free to share your suggestions and opinions. Together, we will create an outstanding translation!

**Comment by n1ghter** [comment]
Hi. Join the https://discord.gg/fQupxC5kyY, we are localizing the game there since May 13 2020 (https://github.com/defterai/StarCitizenModding)
